### ISSUE TITLE
Use viewBox instead of width&height

### ISSUE THREAD

**Issue by analogic** [chillerlan/php-qrcode#30]
When you have variable data length and want same size of inline svg image then it is better to have just viewbox defined. The only quick solution I have come up is ugly hack like this:

```
...
$output = $qr->render($data);
$output = preg_replace('~width="(\d+)px" height="(\d+)px"~', 'style="height: 100%; width: 100%" viewBox="0 0 $1 $2"', $output);
return $output;
```

**Comment by codemasher** [comment]
Seems reasonable. I have now removed the `$scale` option for SVG and instead added `$svgViewBoxSize` which defines a numerical value for width/height of the `viewBox` attribute. If omitted it will assume the qr module count for viewBox width/height. I've also added a `style` attribute with `width: 100%%; height: auto;`. The `width` and `height` attributes have been removed and i put the SVG header in a separate property so that one could inherit the `QRMarkup` class and overwrite its value.
